### DISCUSSION TITLE
Will TrainingSession be deprecated together with ORTTrainer?

### DISCUSSION THREAD

**Discussion by CapJunkrat** [microsoft/onnxruntime#14554] [Other Q&A] [answered]
Since TrainingSession is mainly used in ORTTrainer(yes? no?) and ORTTrainer will be deprecated in version 1.14.  Will TrainingSession  be deprecated soon? If not, where will it be mainly used at?

**Comment by ytaous** [comment, marked as answer]
Yes, per https://github.com/Microsoft/onnxruntime/pull/13022 we are deprecating orttraining from 1.14.
However, TrainingSession would stay for a while as we are still using it for testing purpose.

**Reply by CapJunkrat** [reply]
Great! Thanks!
